hazelcast/src/test/java/com/hazelcast/multimap/MultiMapContainerStatisticsTest.java

```diff
@@ -148,11 +149,45 @@ public void testMultiMapContainerStats() {
         assertSameLastUpdateTimeOnBackup();
     }
 
+    @Test
+    public void testMultimapContainerSize() {
+        //Test put operation
+        assertSize(0);
+        multiMap.put(key, "value1");
+        sleepMillis(10);
+        assertSize(1);
+
+        multiMap.put(key, "value2");
+        multiMap.put(key, "value3");
+        sleepMillis(10);
```

Comment:
What is the added value of the sleep? When the put completes, the size should have been set correctly.